src/sentry/utils/samples.py

```diff
@@ -81,6 +90,19 @@ def load_data(platform, default=None):
         "method": "GET"
     }
 
+    now = datetime.utcnow()
+
+    # Make breadcrumb timestamps relative to right now so they make sense
+    breadcrumbs = data.get('sentry.interfaces.Breadcrumbs')
+    if breadcrumbs is not None:
+        duration = 1000
+        values = breadcrumbs['values']
+        for value in reversed(values):
+            value['timestamp'] = milliseconds_ago(now, duration)
```

Comment:
I think this should work, instead of using the milliseconds_ago function.

```import time

value['timestamp'] = time.mktime((now - timedelta(milliseconds=duration)).timetuple())```